htmresearch/frameworks/nlp/classification_model.py

```diff
@@ -88,6 +88,10 @@ def testModel(self, index, numLabels):
     raise NotImplementedError
 
 
+  def getKNN(self):
+    raise NotImplementedError
```

Comment:
Why not do `return self.classifier` here and eliminate all the redundant subclass methods?